ClientRuntimes/NodeJS/test/data/swaggerPetstore.js

```diff
@@ -36,7 +36,10 @@ var SwaggerPetstore = ( /** @lends SwaggerPetstore */ function() {
    * {@link https://github.com/request/request#requestoptions-callback Options doc}
    */
   function SwaggerPetstore(baseUri, options) {
-    SwaggerPetstore['super_'].call(this, options);
+    if (!options) {
```

Comment:
This should be done by code-gen change happening in the hydra repo. I am tweaking it just to get CI into a 'passing' state